### PR TITLE
Create wmt07.md

### PR DESCRIPTION
**Pull request by liashahnazaryan**
# Description

Created the WMT07 article to add details about the event, including the schedule and results. The automatic evaluation results were excluded from the article as the findings conclude the human judgments to be authoritative. Let me know if you think we should add automatic evaluation results as well, @cefoo!

## Type of PR

- Creates the article _[WMT07]_

### Checklist:

- [x] I have read the [contributing guidelines](contributing.md).
- [x] I have followed the [style guide](http://machinetranslate.org/style).

### CODE REVIEW

File: events/wmt.md [hunk 1]
```diff
@@ -129,45 +129,52 @@ Some people have been organisers over many years:
 
 ### Average score and average z-score
 
-For the **average score**, human assessment scores for translations are standardised according to each human assessor's overall mean and standard deviation.  Then a system-level score is computed.
+For the **average score**, human assessment scores for translations are standardised according to each human assessor's overall mean and standard deviation. Then a system-level score is computed.
```
Comment (cefoo) [inline events/wmt.md:132]: Hi @liashahnazaryan!
Please remember that we separate sentences with an `enter`, following our [style guide](https://machinetranslate.org/style#spacing).
Let me know if you have any question.

Comment (liashahnazaryan) [reply events/wmt.md]: Done. Don't really know how that happened this time :)

**Comment by cefoo** [comment]
> The automatic evaluation results were excluded from the article as the findings conclude the human judgments to be authoritative.

It would be great to include this note in the article. :)

**Comment by liashahnazaryan** [comment]
> > The automatic evaluation results were excluded from the article as the findings conclude the human judgments to be authoritative.
> 
> It would be great to include this note in the article. :)

Done

**Pull request merged**